test/Util.java

```diff
@@ -199,4 +204,52 @@ public static ProcessExecutor createProcessExecutor(Path directory) {
                 .readOutput(true)
                 .destroyOnExit();
     }
+
+    @CommandLine.Command(name = "java")
+    private static class CLIOptions {
+        @CommandLine.Parameters String mainClass;
```

Comment:
The class that is being run during execution, e.g. `com.google.testing.junit.runner.BazelTestRunner`. We need to capture this in the command otherwise picocli thinks it has been passed an unrecognised parameter.